src/Sql/Billing/Stored Procedures/ProviderInvoiceItem_Create.sql

```diff
@@ -7,7 +7,8 @@ CREATE PROCEDURE [dbo].[ProviderInvoiceItem_Create]
     @PlanName NVARCHAR (50),
     @AssignedSeats INT,
     @UsedSeats INT,
-    @Total MONEY
+    @Total MONEY,
+    @Created DATETIME2 (7)
```

Comment:
This would be a breaking change with EDD as we would not be able to deploy this change independent of the code change, trying to call this stored procedure without the extra parameter. The pattern so far has been to create a new version of the procedure and call that with the new release of the code. With a blue green release this would allow the app servers blue green versions to work against the single db. @withinfocus thoughts?

Reply:
@rkac-bw That makes sense. The strange thing about this one is that, in the current state, there actually is no code invoking this SPROC. All of that code is added as part of this PR, so I thought I might be able to just re-create the SPROC as is. That being said, I did forget that the code in this PR would be deployed independently of the migration so if something goes South with the DB migration, the code here would be invoking the old version of the SPROC. I can update these to use V2s.

Reply:
@rkac-bw that's indeed a solution, albeit more advanced. We can just add a `= NULL` and this would be fine.

Reply:
Much cleaner and easier, could even have default value of GETUTCDATE(), either is good by me

Reply:
Updated to make `@Created` default to `NULL` for both the updated "Create" and "Update" stored procedures. You can't default a SP parameter to `GETUTCDATE`, so I set it with `COALESCE` in the SP body.

https://github.com/bitwarden/server/pull/4178/commits/b7704dfddda36f6e344daca3ee7bde631fc2d21a